.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
I love spaces. But when I use 2 spaces, I can't see some parts of code.
I always use 4 spaces instead and I'm very happy :smile: 

I don't understand argumentation about 2 spaces. Any better idea, why 2 instead 4 :question: